src/utils/generator/MaxwellVelocityAssigner.h

```diff
@@ -6,26 +6,26 @@
 
 #include "VelocityAssignerBase.h"
 
-/** The MaxwellVelocityAssigner can be used to assign maxwell boltzmann distributed velocity vectors mathching to a given temperature.
+/** The MaxwellVelocityAssigner can be used to assign maxwell boltzmann distributed velocity vectors matching a given temperature.
  */
 class MaxwellVelocityAssigner : public VelocityAssignerBase {
 public:
-	MaxwellVelocityAssigner(double T = 0, int seed = 0) : VelocityAssignerBase(T), _mt(seed), _normalDistrubtion(0.0, 1.0) {}
+	MaxwellVelocityAssigner(double T = 0, int seed = 0) : VelocityAssignerBase(T), _mt(seed), _normalDistribution(0.0, 1.0) {}
 	~MaxwellVelocityAssigner() {}
 
 	void assignVelocity(Molecule *molecule) {
 		double v_abs = sqrt(/*kB=1*/ T() / molecule->component()->m());
```

Comment:
did you forget to change this line?

Reply:
You mean line 17?
No, actually I made further tests with the MaxwellVelocityAssigner and it seems like the calculation of the absolute temperature is correct as it is.
When I tested it on Friday evening, I added the line suggested in the issue. Nevertheless, I made a mistake by dividing by "3" instead of "3.", because getRotationalDegreesOfFreedom() returns an integer. This results in the whole term being 0 instead of 0.6667... which then gave the correct result.

Anyway: Line 17 in the MaxwellVelocityAssigner is correct as it is. It seems like the rotational degrees of freedom do not need to be considered.